app/main.py

```diff
@@ -14,4 +14,10 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    count = 0
+
+    for character in phrase:
+        if character.lower() == letter.lower():
+            count += 1
+
+    return count
```

Comment:
The solution is ok, but can you try to rewrite it using just 1 line?
Also, use more informative names for variables - not "count"